guides/release/components-update/conditional-content.md

```diff
@@ -0,0 +1,120 @@
+You can conditionally include some content in the output of a template by using
+Ember's `if` syntax in templates.
+
+Let's take a look at two similar components representing a user's username.
+
+```handlebars {data-filename="app/components/received-message/username.hbs"}
+<h4 class="username">
+  Tomster
+  <span class="local-time">their local time is 4:56pm</span>
+</h4>
+```
+
+```handlebars {data-filename="app/components/sent-message/username.hbs"}
+<h4 class="username">Zoey</h4>
+```
+
+We can use arguments to make these two components dynamic, but the first
+username has extra information in the form of the local time of the user.
+
+Let's say we tried to create a single `address` component.
+
+```handlebars {data-filename="app/components/username.hbs"}
+<h4 class="username">
+  {{@name}}
+  <span class="local-time">their local time is {{@localTime}}</span>
+</h4>
+```
+
+If the `<Username>` tag doesn't specify a `@localTime` argument, we'll end up
+with some extra unneeded text in the output.
+
+What we need is a way to only include the local time text `@locaTime` exists at
+all. We can do this with an `if`:
+
+```handlebars {data-filename="app/components/username.hbs"}
+<h4 class="username">
+  {{@name}}
+  {{#if @localTime}}
+    <span class="local-time">their local time is {{@localTime}}</span>
+  {{/if}}
+</h4>
+```
+
+This the syntax for conditionals in an Ember template. The `#if` means that the
+part after the `{{#if ...}}` is nested inside of the conditional. Just like HTML
+tags continue until closed (`<div>` continues until `</div>`), the content
+nested inside an `#if` continues until `{{/if}}`.
+
+## Inline `if`
+
+Sometimes, you need to place conditional content inside an argument or
```

Comment:
This might be more convincing/a smoother transition if we explained _when_ we might want to do this. Or, if we had a justification to use an inline `if` in our example component.